src/Components/App/App.js

```diff
@@ -2,9 +2,7 @@ import React, { useState, useEffect } from 'react';
 import apiCalls from '../../apiCalls';
 import { Redirect } from 'react-router-dom';
 import './App.css';
-import Header from '../Header/Header'
```

Comment:
The refactoring of Header & Nav into a single place rather than in each component clears up code & prevents confusion around something might be breaking.